public/scripts/editor/js/fc/bramble-popupmenu.js

```diff
@@ -0,0 +1,47 @@
+define(["jquery", "fc/bramble-keyhandler", "fc/bramble-underlay"], function($, KeyHandler, Underlay) {
```

Comment:
nit: I'm not sure if it makes a huge difference, but I encouraged Gideon to use `var q = require('q')` style requiring in my reviews of his front-end patches. This is why you see both styles. I figured it might make the switch to a browserified script a little easier down the line.